extensions/file/src/main/java/org/mule/extension/file/internal/FileOperations.java

```diff
@@ -162,14 +162,15 @@ public void write(@UseConfig FileConnectorConfig config, @Connection FileSystem
    * @param targetPath              the target directory where the file is going to be copied
    * @param createParentDirectories whether or not to attempt creating any parent directories which don't exists.
    * @param overwrite               whether or not overwrite the file if the target destination already exists.
+   * @param renameTo                the new file name, {@code null} if the file doesn't need to be renamed
```

Comment:
we need to shift the wording on the jdocs to be less technical and more user oriented. Rephrase this explaining that if provided, the copy will have this new name. Otherwise, the current name will be honoured.